mappings/net/minecraft/util/Util.mapping

```diff
@@ -205,6 +205,11 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 		COMMENT @see #combine(List)
 		COMMENT @see #combineSafe(List)
 		ARG 0 futures
+	METHOD method_43498 waitUntilDone (Ljava/util/function/Function;Ljava/util/function/Predicate;)Ljava/lang/Object;
```

Comment:
This isn't just waiting; this is running tasks created when the function is called with an executor that add runnables to a queue for the current main thread to run. So, this can just run prepared tasks on the current main thread once any is done, to reduce computational waste. I recommend names like
```suggestion
	METHOD method_43498 waitAndApply (Ljava/util/function/Function;Ljava/util/function/Predicate;)Ljava/lang/Object;
```